MAVProxy/modules/mavproxy_paramedit/__init__.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+'''
+param editor module
+Akshath Singhal
+June 2019
```

Comment:
Need a copyright here

Reply:
we're not doing it for other modules, we can assume the top level license here I think